Formula/chain-bench.rb

```diff
@@ -0,0 +1,20 @@
+class ChainBench < Formula
+  desc "Software supply chain auditing tool based on CIS benchmark"
+  homepage "https://github.com/aquasecurity/chain-bench"
+  url "https://github.com/aquasecurity/chain-bench/archive/v0.1.3.tar.gz"
+  sha256 "46b63528ddd0e09b65539a69aefbf4229343e636854d0dd5a241870717843423"
+  license "Apache-2.0"
+  head "https://github.com/aquasecurity/chain-bench.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w -X=main.version=#{version}"), "./cmd/chain-bench"
+  end
+
+  test do
+    assert_match(/.*Fetch\sStarting.*/, shell_output("#{bin}/chain-bench scan", 1))
```

Comment:
I think this'll work like this too
```suggestion
    assert_match("Fetch Starting", shell_output("#{bin}/chain-bench scan", 1))
```

Reply:
Fixed it with the latest changes